modules/benchmark/R/calc.benchmark.R

```diff
@@ -11,21 +11,29 @@
 
 calc.benchmark <- function(settings, bety) {
   
-  library(RPostgreSQL)
-  library(data.table)
+  # dplyr functions
+  tbl     <- dplyr::tbl
+  filter  <- dplyr::filter
+  rename  <- dplyr::rename
+  collect <- dplyr::collect
+  select  <- dplyr::select
   
```

Comment:
Not sure I like this, especially with things like year which is to easy a variable to use in the code as well and get strange compile/runtime errors. This is one problem I have in general with non strongly typed programming languages. Not sure if something like lintr would catch these type of errors (redeclaring a variable to a different type).